app/models/manageiq/providers/inventory/persister/builder/container_manager.rb

```diff
@@ -104,10 +103,9 @@ def container_image_registries
 
         def container_groups
           add_properties(
-            :secondary_refs         => {:by_container_project_and_name => %i[container_project name]},
-            :attributes_blacklist   => %i[namespace],
-            :delete_method          => :disconnect_inv,
-            :custom_reconnect_block => custom_reconnect_block
```

Comment:
Is this block a custom hook that will only be called if it exists?  Do we need to remove the caller or do we have remaining instances of `custom_reconnect_block`?

Reply:
Hm not sure what you mean, if `collection.custom_reconnect_block` is `nil` then `inventory_refresh` won't call anything for that collection.

Reply:
Right, was this the only one?  If so, the caller doesn't need to even call it.

Reply:
Oh, no definitely not the only one we still use it in the infra_manager for hosts/vms.